spec/acceptance/docker_spec.rb

```diff
@@ -1,12 +1,12 @@
 require 'spec_helper_acceptance'
 
-describe 'docker class' do
+describe 'docker' do
   case fact('osfamily')
   when 'RedHat'
-    if fact('operatingsystemrelease').to_f >= 7
-      package_name = 'docker'
+    package_name = if fact('operatingsystemrelease').to_f >= 7
```

Comment:
I don't like if-stmts as values. I'd prefer either the original version or using a ternary. I'd defer to "local style", tho.